src/utils/getFileTypeFromPath.ts

```diff
@@ -0,0 +1,33 @@
+import { FileType } from '../tree/types';
+
+/** Outputs the file type of a certain path
+ * @param {string} path the path of a specific file or folder
+ * @returns {FileType} file type such as .FILE, .CONFIG_FILE, or .FOLDER
+ * @description using inputted relative GitHub path, get file type of path
+ */
+
+export const getFileTypeFromPath = (path: string): FileType => {
+  // Count amount of periods assuming the path is relative and starts with "./"
```

Comment:
I just noticed something. 
The path which the API produces doesn't start with "./"
So if the clickable link would be something like [components](./src/components) but the address we get from API as path is going to be "src/components"

Reply:
See that's why you need to check :)
Becuase if you just get 
src -> that becomes [src](./src)

Reply:
Thank you, I've fixed this!

Reply:
Hey @ralph-dev, mind checking over the function file and test case file to see if it's ready for merge? I've updated them both and changed the structure of the function so it's more adaptive at catching invalid paths.